### ISSUE TITLE
Zip files sould be removed from the repo

### ISSUE THREAD

**Issue by mrtommyb** [spacetelescope/roman-technical-information#31]
We have some zipfile in the repo - I'm not sure that this makes sense, and we should expand these into the individual files.
An example is: data/WideFieldInstrument/Imaging/EffectiveAreas

**Comment by tddesjardins** [comment]
Makes sense to me. Do you want to add further subdirectories in situations like that, or do you think it's better to just label the files appropriately? Personally, I'm more in favor of flat directories when possible.

**Comment by mrtommyb** [comment]
Yes, I prefer a flatter structure too.
